### DISCUSSION TITLE
Is this and puppeteer similar?

### DISCUSSION THREAD

**Discussion by NCLnclNCL** [kaliiiiiiiiii/Selenium-Driverless#250] [Q&A] [answered]
I mean it uses the same cdp, so what's the difference, puputeer can also bypass cloudflare, bet365 by filtering targets on invalid urls(from frame)

**Comment by ttlns** [comment, marked as answer]
Not sure what you mean with
> by filtering targets on invalid urls(from frame)

But pretty sure that puppeteer has other leaks as well (not just the `Runtime.enable` one) 

In any case, puppeteer has a different structure than driverless and isn't really similar in that sense
